packages/design-picker/src/generated-designs-config.json

```diff
@@ -0,0 +1,43 @@
+[
+	{
+		"slug": "business",
+		"title": "Business",
+		"recipe": {
+			"theme": "pub/winkel",
```

Comment:
Oh, I just think maybe this field should be `stylesheet` as the stylesheet is composed of repo and theme (or theme slug). That being said, `stylesheet` = `${repo}/${theme}`. What do you think?

Reply:
Ah, yeah I actually am not sure about this either, as in Calypso and wpcom codebases, we sometimes use `stylesheet` and `theme` to refer to this.

What if we change this to `themeStylesheet`? (And maybe we need to also change the respective query param in the `/block-previews` endpoint 😅 )

Reply:
Changed to `stylesheet` here and in the backend.